feat: link crawl-related resources, robots.txt, HTTP headers, HTML meta (#39991)
mdn/content: files/en-us/glossary/robots.txt/index.md

{{GlossarySidebar}}

A **robots.txt** is a file that is usually placed in the root of a website (for example, `https://www.example.com/robots.txt`).

> It specifies whether {{Glossary("crawler", "crawlers")}} are allowed or disallowed from accessing an entire website or to certain resources on a website.

Reviewer: ```suggestion
It specifies whether or not {{Glossary("crawler", "crawlers")}} are allowed access to an entire website, or to specified resources.
```
